WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.java

```diff
@@ -934,4 +829,61 @@ void pauseWebView() {
         }
     }
 
+    @Override
+    public void onScrollUp() {
+        showToolbar(true);
+        showFooter(true);
+    }
+
+    @Override
+    public void onScrollDown() {
+        if (mScrollView.canScrollDown()
+                && mScrollView.canScrollUp()
+                && mScrollView.getScrollY() > mToolbarHeight) {
+            showToolbar(false);
+            showFooter(false);
+        }
+    }
+
+    @Override
+    public void onScrollCompleted() {
+        if (!mScrollView.canScrollDown()) {
+            showToolbar(true);
+            showFooter(true);
+        }
+    }
+
+    private void showToolbar(boolean show) {
+        if (mAutoHideToolbarListener != null) {
+            mAutoHideToolbarListener.onShowHideToolbar(show);
+        }
+    }
+
+    private void showFooter(boolean show) {
+        if (isAdded() && canShowFooter()) {
+            AniUtils.animateBottomBar(mLayoutFooter, show);
+        }
+    }
+
+    /*
+     * can we show the footer bar which contains the like & comment counts?
+     */
+    private boolean canShowFooter() {
+        return canShowLikeCount() || canShowCommentCount();
+    }
+
+    private boolean canShowCommentCount() {
+        return mPost != null
+                && mPost.isWP()
+                && !mPost.isJetpack
```

Comment:
> If the post isWP() does that imply !isJetpack?

That used to be the case, but not any more. The short explanation is that there are two sources for the Reader: WordPress blogs and external RSS feeds. In the past Jetpack blogs were treated like RSS, but recent changes have started treating them like WP blogs so we can poll them more efficiently and support likes. Comments are still a WIP.